api/v1alpha1/database_types.go

```diff
@@ -33,11 +33,12 @@ type DatabaseSpec struct {
 
 // DatabaseStatus defines the observed state of Database
 type DatabaseStatus struct {
-	IPAddress        string `json:"ipAddress"`
-	Id               string `json:"id"`
-	Status           string `json:"status"`
-	DatabaseServerId string `json:"dbServerId"`
-	Type             string `json:"type"`
+	IPAddress               string `json:"ipAddress"`
+	Id                      string `json:"id"`
+	Status                  string `json:"status"`
+	DatabaseServerId        string `json:"dbServerId"`
+	Type                    string `json:"type"`
+	ProvisioningOperationId string `json:"ProvisioningOperationId"`
```

Comment:
Can you please change this to camelCase?

Reply:
Done!